tests_config/run_spec.js

```diff
@@ -38,6 +38,35 @@ const unstableAstTests = new Map(
   })
 );
 
+const astChangedTests = new Map(
```

Comment:
Two questions here:

1. Is the name of this variable accurate?
2. Why is the bytecode comparison only done for these files? Why don't we do it for all of them?

Reply:
Ohh, the bytecode comparison is *only* done for these files, I understand now. I still think the name of the variable is not very descriptive.

Reply:
1. Just changed the name slightly to be more precise.
2. For now I believe AST comparison should be enough and byte-code comparison applies only on the cases when we choose to change the AST.
  I agree that best case would be to ultimately compare byte codes but I also see all of the work needed to do that properly.
  Making sure every test is compilable and also keeping track of major syntax changes across different compilers.